### PR TITLE
[DNVB] Navigation Menu

### PR DESCRIPTION
**Pull request by flair-conor**
Initial commit

#36

### CODE REVIEW

**Comment by flair-conor** [comment]
Will rebase and change the SVGs used once they come in on the main branch. They were no longer on the nas and are already in another PR.

File: templates/theme-components/common/quick-search.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
             <div class="form-field">
                 <label class="is-srOnly" for="search_query">{{lang "search.quick_search.input_label"}}</label>
                 <input class="form-input" data-search-quick name="search_query" id="search_query" data-error-message="{{lang 'search.error.empty_field'}}" placeholder="{{lang 'search.quick_search.input_placeholder'}}" autocomplete="off">
+                <svg class="quickSearch-icon"><use xlink:href="#icon-logo-mastercard"></use></svg>
```
Comment (flair-duncan) [inline templates/theme-components/common/quick-search.html]: ???

Comment (flair-conor) [reply templates/theme-components/common/quick-search.html]: > Will rebase and change the SVGs used once they come in on the main branch. They were no longer on the nas and are already in another PR.

**Comment by flair-duncan** [comment]
Closes #36

**Pull request merged**